makepath_test.go

```diff
@@ -0,0 +1,25 @@
+package pathutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMakePath(t *testing.T) {
```

Comment:
I dont't see tests for MakePathMode method

Reply:
Good point, I do it...

Reply:
becuase `MakePath` use internal method `MakePathMode`